web/src/helpers/fetch.ts

```diff
@@ -1,5 +1,13 @@
 import { getCSRFToken } from 'web/src/helpers/CSRFToken';
 
+function isJSON(str: string) {
```

Comment:
```suggestion
const isJSON = (str: string) => {
```

Сделай плиз такой стиль